.docker/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM php:7.2-cli-alpine
+
+RUN apk --update add wget \
+    php7-curl
+
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer
+
+RUN mkdir -p /var/www
+
+WORKDIR /var/www
+
+COPY . /var/www
+
+VOLUME /var/www
+
+CMD ["/bin/sh"]
```

```suggestion
CMD ["php -S localhots:$PORT"]
```

Com essas sugestões, por padrão, o container sobre expondo a porta 8080, ficando a critério do desenvolvedor modificar esse comportamento caso necessário.

---

Entendi, mas há necessidade de expor a porta? Porque como é uma biblioteca, o intuito do container é só ter o PHP e composer para instalar dependências e executar os testes com PHPUnit.

---

Verdade, me passei.